src/raps/registerENS.ts

```diff
@@ -127,14 +131,26 @@ export const createTransferENSRap = async (
   } = ensActionParameters;
 
   if (clearRecords) {
-    const records = await fetchProfileRecords(ensActionParameters.name);
+    const [allRecords, allCoinAddresses] = await Promise.all([
+      fetchRecords(ensActionParameters.name, {
+        supportedOnly: false,
+      }),
+      fetchCoinAddresses(ensActionParameters.name, {
+        supportedOnly: false,
+      }),
+    ]);
     const emptyRecords = Object.keys({
-      ...(records.coinAddresses || {}),
-      ...(records.records || {}),
+      ...(allCoinAddresses || {}),
+      ...(allRecords || {}),
     }).reduce((records, recordKey) => {
+      let value = '';
+      // Use empty address for ETH record as an empty string throws error
+      if (recordKey === ENS_RECORDS.ETH) {
+        value = emptyAddress;
+      }
       return {
         ...records,
-        [recordKey]: '',
+        [recordKey]: value,
```

Comment:
imo would look cleaner if it was `[recordKey]: recordKey === ENS_RECORDS.ETH ? emptyAddress : ''` but up to u